azure-pipelines.yml

```diff
@@ -19,9 +19,7 @@ pr:
       - LICENSE.txt
       - dco.txt
       - .github/ISSUE_TEMPLATE/*.md
-      - .github/labeler.yml
       - .github/boring-cyborg.yml
-      - .github/workflows/*
```

Comment:
why are you removing that one? it will show up eventually :)

Reply:
Ah I thought you replaced with the boring cyborg one. I'll revert that change